packages/react-composites/src/composites/common/BaseComposite.tsx

```diff
@@ -93,7 +102,9 @@ export const BaseProvider = (
   const CompositeElement = (
     <FluentThemeProvider fluentTheme={fluentTheme} rtl={rtl}>
       <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0" />
-      <Customizer scopedSettings={{ Layer: { hostId: globalLayerHostId } }}>{props.children}</Customizer>
+      <Customizer scopedSettings={{ Layer: { hostId: globalLayerHostId } }}>
```

Comment:
tangential: What is `Customizer`? That's a very enigmatic and not very useful name.

Reply:
@PorterNan Might know more about this. Customizer was already here.